tracing/jaeger/logging.go

```diff
@@ -0,0 +1,21 @@
+package jaeger
+
+import (
+	"context"
+
+	"github.com/rs/zerolog"
+
+	"github.com/kiali/kiali/log"
+)
+
+// getLoggerFromContextGRPCJaeger returns the logger found in the context and adds the type key to refer to this as the gRPC Jaeger logger
+func getLoggerFromContextGRPCJaeger(ctx context.Context) *zerolog.Logger {
+	zl := log.FromContext(ctx).With().Str("impl", "gRPC Jaeger").Logger()
```

Comment:
`impl` is kind of vague. I'm guessing this is shorthand for `implementation` but a user parsing the logs may not know what this means. Maybe "tracing-type"? Or just "client" since we already have "group = tracing" from the parent context? Do we even need this context added or is "group = tracing" from the parent good enough?

Reply:
group=tracing is not enough.. this shows the log came from one of our implementations (there are 4 of them - jaeger (http, grpc) and tempo (http, grpc).). Does a user care? These are for us, not them :)

Reply:
renamed `impl` to `client-type` which better describes what this is

Reply:
> Does a user care? These are for us, not them

It's for both. Users should be able to more easily parse their logs with the added context _and_ be able to submit those logs in issues for us to more easily analyze.